sdk/typescript/src/types/common.ts

```diff
@@ -9,3 +9,35 @@ export type ObjectOwner =
   | { ObjectOwner: SuiAddress }
   | 'Shared'
   | 'Immutable';
+
+
+const TX_DIGEST_BASE64_LENGTH = 44;
+// taken from https://rgxdb.com/r/1NUN74O6
+const VALID_BASE64_REGEX =
+  /^(?:[a-zA-Z0-9+\/]{4})*(?:|(?:[a-zA-Z0-9+\/]{3}=)|(?:[a-zA-Z0-9+\/]{2}==)|(?:[a-zA-Z0-9+\/]{1}===))$/;
+
+export function isValidTransactionDigest(value: string): value is TransactionDigest {
+  return value.length === TX_DIGEST_BASE64_LENGTH
```

Comment:
after the change from 44 -> 32, you can do 
```suggestion
  // need to add `import { Base64DataBuffer } from '../serialization/base64';`
  return new Base64DataBuffer(value).getLength() === TX_DIGEST_LENGTH
```